### PR TITLE
[lldb] Annotate swift tests with @swiftTest

### PR DESCRIPTION
**Pull request by bulbazord**

### CODE REVIEW

**Review by kastiglione — COMMENTED** [review]
why did you inline `setUp` into each of the test functions?

**Comment by bulbazord** [comment]
Because `setUp` invokes `self.build()` which will fail without swift enabled. Even if the tests are marked `@swiftTest` the setup will run regardless.

**Comment by kastiglione** [comment]
sounds like a bug (why setup a test that won't run?) – but ok. thanks

File: lldb/test/API/commands/expression/persistent_result/swift/TestSwiftPersistentResult.py [hunk 1]
```diff
@@ -11,29 +11,45 @@
 class TestCase(TestBase):
     def setUp(self):
         TestBase.setUp(self)
```
Comment (kastiglione) [inline lldb/test/API/commands/expression/persistent_result/swift/TestSwiftPersistentResult.py]: may as well delete this then

**Review by kastiglione — APPROVED** [review]

File: lldb/test/API/commands/expression/persistent_result/swift/TestSwiftPersistentResult.py (hunk 1 above)
Comment (bulbazord) [reply lldb/test/API/commands/expression/persistent_result/swift/TestSwiftPersistentResult.py]: Good point

**Comment by bulbazord** [comment]
@swift-ci please test

**Pull request merged**